wire/Makefile

```diff
@@ -50,11 +51,15 @@ wire/extracted_onion_experimental_csv:
 	@touch $@
 
 ifeq ($(EXPERIMENTAL_FEATURES),1)
-wire/gen_peer_wire_csv: wire/extracted_peer_experimental_csv wire/extracted_peer_wire_csv
-	@patch --silent -o $@ wire/extracted_peer_wire_csv $<
+wire/gen_peer_wire_csv: wire/extracted_peer_wire_csv
+	@cp $< $@
+	@$(eval EXP_FILES := $(wildcard wire/extracted_peer_experimental_*))
```

Comment:
uses this instead of an inline shell glob because empty globs return themselves, which errors.

Reply:
True, and shopt -s nullglob is a bashism :(

Still, using a Makefile var instead of using $(wildcard...) directly below is kinda weird.

Also, please consider GitHub to be transient: anyhing worth commenting like this needs to be in the source itself...